collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -176,6 +176,22 @@ private void uploadForms(Context context) {
         }
     }
 
+    private boolean submitInstance(String jrFormId, boolean autoSendSettings) {
```

Comment:
Also, it would be useful to have some comments that indicate that if the form explicitly sets the auto-send property, then it overrides the preferences. The general settings value is only used if auto-send isn't set at the form level. That will make it easier to verify the intent of the code in the future.